assets/os2forms_payment.css

```diff
@@ -0,0 +1,7 @@
+#testHallo--description {
+  font-size: 20px !important;
+}
+
```

Comment:
This looks like leftovers from a debugging session.

```suggestion
```

Reply:
Right. Deleted.